test/fixtures/systemd-unit.yml

```diff
@@ -0,0 +1,56 @@
+# syntax=local/dalec/frontend
+
+name: dalec-test-systemd-unit
+description: A test fixture which tests the installation of a simple systemd unit
+website: https://www.github.com/Azure/dalec
+version: 0.0.1
+revision: 1
+vendor: Microsoft
+license: Apache 2.0
+packager: Microsoft <support@microsoft.com>
+
+targets:
+  mariner2:
+    image:
+      base: mcr.microsoft.com/cbl-mariner/base/core:2.0
```

Comment:
How come this needs an explicit base image?

Reply:
It actually does not